var/spack/repos/builtin/packages/libsonata/package.py

```diff
@@ -15,10 +15,10 @@ class Libsonata(CMakePackage):
     See also:
     https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md
     """
-    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/common/libsonata"
-    url      = "ssh://bbpcode.epfl.ch/common/libsonata"
```

Comment:
just fyi : `url` typically points to released tarball and it's useful for commands like `spack checksum`

Reply:
OK, thanks for the clarification.